diff --git a/node_modules/@open-wa/wa-automate/dist/lib/wapi.js b/node_modules/@open-wa/wa-automate/dist/lib/wapi.js
index c256310..3eb6169 100644
--- a/node_modules/@open-wa/wa-automate/dist/lib/wapi.js
+++ b/node_modules/@open-wa/wa-automate/dist/lib/wapi.js
@@ -90,7 +90,7 @@ if (!window.Store||!window.Store.Msg) {
         const parasite = `parasite${Date.now()}`
         // webpackJsonp([], { [parasite]: (x, y, z) => getStore(z) }, [parasite]);
         if (typeof webpackJsonp === 'function') webpackJsonp([], {[parasite]: (x, y, z) => getStore(z)}, [parasite]); 
-        else webpackChunkbuild.push([[parasite], {}, function (o, e, t) {let modules = []; for (let idx in o.m) {modules.push(o(idx));}	getStore(modules);}]);        
+        else webpackChunkwhatsapp_web_client.push([[parasite], {}, function (o, e, t) {let modules = []; for (let idx in o.m) {modules.push(o(idx));}	getStore(modules);}]);        
     })();
 }
 
